views/AppView.js

```diff
@@ -0,0 +1,64 @@
+var AppView = Backbone.View.extend({
+  el: $('body'),
+
+  events: {
+    'click .search': 'handleSearch',
+    'click .view-video': 'viewVideo'
+  },
+
+  mainVideo: null,
+
+  initialize: function () {
+    this.listenTo(this.model.get('videos'), 'add', this.renderVideo);
+    this.listenTo(this.model.get('videos'), 'add', this.setCurrentVideo);
+    this.listenTo(this.model, 'change:show_video', this.renderPage);
+    this.listenTo(this.model, 'change:current_video', this.renderMainVideo);
+    this.$searchInput = this.$('#search-query');
+    this.listenTo(this.model.get('videos'), 'reset', this.renderVideos);
+    this.listenTo(this.model, 'change:searchQuery', this.updateSearchUrl);
+    this.renderVideos();
+  },
+
+  setCurrentVideo: function () {
+    var currentVideo = appModel.attributes.videos.at(0);
```

Comment:
We don't ever want to access `attributes` directly, but instead to use `get`.

Also, you normally wouldn't have access to the `appModel` variable (we'll get to that with JavaScript Modules later). Instead use `this.model`.